rootfs/etc/profile.d/prompt.sh

```diff
@@ -5,7 +5,23 @@ shopt -s checkwinsize
 # Cache the current screen size
 export SCREEN_SIZE="${LINES}x${COLUMNS}"
 
-export PROMPT_COMMAND=prompter
+export PROMPT_COMMAND
+function _install_prompter() {
```

Comment:
Can you add a note what this is doing in "plain english" because not sure I follow what it does/is needed.

Reply:
Done. See https://github.com/cloudposse/geodesic/pull/422/commits/c55f06913a95748472c6e0d50cc5c37339a7b544